Tests/BackendIntegrationTests/Helpers/CustomPuchasesOrchestrator.swift

```diff
@@ -0,0 +1,162 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CustomPuchasesOrchestrator.swift
+//
+//  Created by Nacho Soto on 7/27/23.
+
+@testable import RevenueCat
+import StoreKit
+
+/// A simplified `PurchasesOrchestrator` used for simulating purchases
+/// made from outside the SDK.
+final class CustomPurchasesOrchestrator: NSObject {
```

Comment:
Maybe we could call it `ExternalPurchasesSimulator` or something?

Reply:
👍🏻 
I'll name it `ExternalPurchasesManager` so "simulator" doesn't get confusing.